app/models/project_update_priority.rb

```diff
@@ -0,0 +1,16 @@
+class ProjectUpdatePriority < ApplicationRecord
+  belongs_to :project
+  enum priority: [ :low, :medium, :high ]
+
+  def self.low_priority_date
+    1.month.ago
+  end
+
+  def self.medium_priority_date
+    2.weeks.ago
+  end
+
+  def self.high_priority_date
```

Comment:
I think we should be careful utilizing these dates per priority. We may end up in a situation where we can't keep up with the amount of projects in a certain priority if we dump a lot in one priority because the time limit we are checking for is too small.